src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -527,5 +528,14 @@
 					"	send \"You equipped %event-item%!\"")
 				.since("INSERT VERSION");
 		}
+		if (Skript.classExists("org.bukkit.event.block.SpongeAbsorbEvent")) {
+			Skript.registerEvent("Sponge Absorb", SimpleEvent.class, SpongeAbsorbEvent.class, "sponge absorb")
+					.description("Called when an sponge absorbs.")
```

Comment:
When *a* sponge absorbs *water blocks*, right?

Reply:
For now, but who knows what Mojang is planning on the future with the 1.16 update, maybe Lava Sponges? xD